lib/rex/random_identifier/generator.rb

```diff
@@ -70,8 +70,8 @@ class ExhaustedSpaceError < StandardError; end
           const continue debugger default delete do else export extends false finally for function if import in
           instanceof new null return super switch this throw true try typeof var void while with let static yield
           await arguments as async eval from get of set enum implements interface package private protected public
-          abstract boolean byte char double final float goto int long native short synchronized throws transient volatile
-          atob alert blur btoa cancelAnimationFrame cancelIdleCallback clearInterval clearTimeout close confirm
+          abstract boolean byte char double final float goto int long native short synchronized throws transient
+          volatile atob alert blur btoa cancelAnimationFrame cancelIdleCallback clearInterval clearTimeout close confirm
           createImageBitmap dump fetch find focus getComputedStyle getDefaultComputedStyle getScreenDetails getSelection
           matchMedia moveBy moveTo open postMessage print prompt queryLocalFonts queueMicrotask reportError
           requestAnimationFrame requestIdleCallback resizeBy resizeTo scroll scrollBy scrollByLines scrollByPages
```

Comment:
Not a blocker; Looks like this list is missing some extra syntactic values `["break", "case", "catch", "class"]`

Reply:
I can add that. I had originally just changed this line to wrap at 120 chars but happy to add missing words.